packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py

```diff
@@ -1,13 +1,80 @@
+import commands
+import time
+from itertools import chain
 from onl.platform.base import *
 from onl.platform.accton import *
 
+init_ipmi_dev = [
+    'echo "remove,kcs,i/o,0xca2" > /sys/module/ipmi_si/parameters/hotmod',
+    'echo "add,kcs,i/o,0xca2" > /sys/module/ipmi_si/parameters/hotmod']
+
+ATTEMPTS = 5
+INTERVAL = 3
```

Comment:
Why these numbers? Please add a comment why it does not work at first try, and why these values were chosen.